addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -70,15 +70,15 @@ def _prepare_mail_values(self, slide_channel_partner):
         body = self._render_field('body', slide_channel_partner.ids)[slide_channel_partner.id]
         # post the message
         mail_values = {
-            'email_from': self.env.user.email_formatted,
+            'attachment_ids': [(4, att.id) for att in self.attachment_ids],
```

Comment:
You did that [somewhere else on the PR](https://github.com/odoo/odoo/pull/99482/files#diff-4830a288b3a08d110fca5ca4a5a0fe6cc2252b723a8c64ff15fba675d7ac19cfR635) and a little above.

Reply:
That's because it actually fallbacks on message_post with does expect a list of IDs, to ease manipulation (same for attachments). It is afterwards transformed into a list of commands.